plugins/GSdx/GSDeviceDX.h

```diff
@@ -162,7 +154,7 @@ class GSDeviceDX : public GSDevice
 				uint32 fst:1;
 				uint32 wms:2;
 				uint32 wmt:2;
-				uint32 fmt:3;
+				uint32 fmt:4;
```

Comment:
You might need to update the `uint32()` operator. For openGL I added the missing bits as free and static assert on the structure size. The init will set 0 to all bits. And if I forget to reduce the free bits, I will get an assert.